src/actions/transferActions.js

```diff
@@ -202,8 +202,32 @@ async function _submitTx (dispatch, txRequest, accountInfo) {
         })
     } else if (cryptoType === 'dai') {
       const _web3 = new Web3(new Web3.providers.HttpProvider(infuraApi))
-      const amountInWei = _web3.utils.toWei(transferAmount.toString(), 'ether')
-      const signedTransactionObject = await ledgerNanoS.signSendTrasaction(0, getCrypto('dai').address, ERC20_ABI, 'transfer', escrow.address, amountInWei)
+      // send out tx fee for next tx
+      const signedTxFee = await ledgerNanoS.signSendEther(
+        0,
+        escrow.address,
+        txCost.costByType.ethTransfer,
+        {
+          gasLimit: parseInt(txCost.costByType.txCostEth.gas),
+          gasPrice: parseInt(txCost.costByType.txCostEth.price)
+        })
+      txRequest.sendTxFeeTxHash = await new Promise((resolve, reject) => {
```

Comment:
consider using https://github.com/ventureum/Chainsfer/blob/fcac2c23181638a3d2b133491f182fc642f26bdd/src/actions/transferActions.js#L24

Reply:
it uses difference web3 functions.